conf/awsbatch.config

```diff
@@ -1,6 +1,6 @@
 /*
  * -------------------------------------------------
- *  Nextflow config file for AWS Batch
+ *  Nextflow config file for AWS Batch [not used for lncpipe]
```

Comment:
```suggestion
 *  Nextflow config file for AWS Batch
```

Someone might want to use it there, just keep it in ;-)

Reply:
Okey